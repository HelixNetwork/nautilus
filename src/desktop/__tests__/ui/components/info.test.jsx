
import React from 'react';
import { shallow } from 'enzyme';

import Info from 'ui/components/info';

const props = {
    children: 'Lorem ipsum',
};

describe('Test for Info component', () => {
    test('Render the Info component', () => {
        const wrapper = shallow(<Info {...props} />);

        expect(wrapper).toMatchSnapshot();
    });

    test('Testing the Info content', () => {
        const wrapper = shallow(<Info {...props} />);

        expect(wrapper.text()).toEqual('<Icon />Lorem ipsum');
    });
});
