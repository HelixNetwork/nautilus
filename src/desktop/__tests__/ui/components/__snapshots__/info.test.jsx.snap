// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Info component Render the Info component 1`] = `
<div
  className="wrapper"
>
  <Icon
    icon="info"
    size={28}
  />
  Lorem ipsum
</div>
`;
